File: acmicpc/단계별로 풀어보기/for문 사용해보기/11720.c
#include <stdio.h>

int main() {
	int N,input,sum = 0;
	int i;
	scanf("%d", &N);
	for (i = 0; i < N; i++) {
		scanf("%1d", &input);
		sum += input;
	}
	printf("%d", sum);
	return 0;
}

File: acmicpc/단계별로 풀어보기/문자열 사용하기/11720.java
// 숫자의 합

import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;

public class Java_11720 {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        int N = Integer.parseInt(br.readLine());
        String[] Input = br.readLine().split("");
        int sum=0;
        for(int i = 0; i < N; i++) {
            sum += Integer.parseInt(Input[i]);
        }
        System.out.println(sum);
    }
}
